examples/jenkins/pipeline/README.md

```diff
@@ -0,0 +1,60 @@
+This set of files will allow you to deploy a Jenkins server that is capable of executing Jenkins pipelines and
+utilize pods run on OpenShift as Jenkins slaves.
+
+To walk through the example:
+1) Stand up an openshift cluster from origin master, installing the standard imagestreams to the openshift namespace:
+
+https://github.com/openshift/origin
```

Comment:
well i was more trying to point them to origin since the step is "stand up origin" but i can add a link to the imagestreams as well.